src/attributes.jl

```diff
@@ -86,21 +86,31 @@ end
 
 Return a `Bool` indicating whether it is possible to query attribute `attr` from the solver `s`.
 
-    cangetattribute(m::AbstractSolverInstance, attr::AbstractVariableAttribute, R::Type{VariableReference})::Bool
-    cangetattribute(m::AbstractSolverInstance, attr::AbstractConstraintAttribute, R::Type{ConstraintReference{F,S})::Bool
+    cangetattribute(m::AbstractSolverInstance, attr::AbstractVariableAttribute, v::VariableReference)::Bool
+    cangetattribute(m::AbstractSolverInstance, attr::AbstractConstraintAttribute, c::ConstraintReference{F,S})::Bool
+
+Return a `Bool` indicating whether the solver instance `m` currently has a value for the attributed specified by attribute type `attr` applied to the variable reference `v` or constraint reference `c`.
 
-Return a `Bool` indicating whether the solver instance `m` currently has a value for the attributed specified by attribute type `attr` applied to the reference type `R`.
+    cangetattribute(m::AbstractSolverInstance, attr::AbstractVariableAttribute, v::Vector{VariableReference})::Bool
+    cangetattribute(m::AbstractSolverInstance, attr::AbstractConstraintAttribute, c::Vector{ConstraintReference{F,S}})::Bool
+
+Return a `Bool` indicating whether the solver instance `m` currently has a value for the attributed specified by attribute type `attr` applied to *every* variable references in `v` or constraint reference in `c`.
 
 ### Examples
 
 ```julia
 cangetattribute(m, ObjectiveValue())
 cangetattribute(m, VariablePrimalStart(), varref)
 cangetattribute(m, ConstraintPrimal(), conref)
+cangetattribute(m, VariablePrimal(), [ref1, ref2])
 ```
 """
 function cangetattribute end
 cangetattribute(m::AbstractSolverInstance, attr::AnyAttribute) = false
+cangetattribute(m::AbstractSolverInstance, attr::AnyAttribute, ref::AnyReference) = false
+function cangetattribute(m::AbstractSolverInstance, attr::AnyAttribute, refs::Vector{T}) where T <: AnyReference
```

Comment:
You can write `Vector{<: AnyReference}` directly without needing to create the variable `T`. I think it is new in Julia v0.6 and it is quite handy :-P